Formula/scm-manager.rb

```diff
@@ -25,25 +25,13 @@ def install
 
     libexec.install Dir["*"]
 
-    (bin/"scm-server").write <<~EOS
-      #!/bin/bash
-      BASEDIR="#{libexec}"
-      REPO="#{libexec}/lib"
-      export JAVA_HOME=$(#{Language::Java.java_home_cmd("1.8")})
-      "#{libexec}/bin/scm-server" "$@"
-    EOS
-    chmod 0755, bin/"scm-server"
-
-    tools = libexec/"tools"
-    tools.install resource("client")
+    env = Language::Java.overridable_java_home_env("1.8")
```

Comment:
Recomends java 11 and is currently at 2.5.0 upstream

Reply:
It seems they have changed it quite a lot. I would update it in a separate PR

And current version doesn't work with java 11:
```
$ scm-cli-client version
Exception in thread "main" java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException
	at sonia.scm.cli.config.ScmClientConfig.load(ScmClientConfig.java:108)
	at sonia.scm.cli.config.ScmClientConfig.getInstance(ScmClientConfig.java:90)
	at sonia.scm.cli.App.loadConfig(App.java:190)
	at sonia.scm.cli.App.run(App.java:153)
	at sonia.scm.cli.App.main(App.java:127)
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.JAXBException
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	... 5 more
```